src/compas/geometry/surfaces/surface.py

```diff
@@ -51,7 +45,9 @@ class Surface(Geometry):
     """
 
     def __new__(cls, *args, **kwargs):
-        return new_surface(cls, *args, **kwargs)
+        if cls is Surface:
+            raise TypeError("Instantiating the base Surface class directly is not allowed.")
```

Comment:
I would maybe change the wording a bit. the caller doesn't necessarily know an entirely different class is gonna be created, that's abstracted by the plugin system. if I saw something like this I'd go looking for the right child class to instantiate myself. Instead, they should know the default constructor is not allowed but the other ones yes.